<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePointLayerImportRaster</class>
 <widget class="QDialog" name="CreatePointLayerImportRaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>saniHUB RedBasica</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name of node layer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Select the raster layer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="txtNodeLayertName"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="cboRasterLayer"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
